packages/app/src/composables/useRelevantRun.ts

```diff
@@ -1,20 +1,53 @@
 import { gql, useSubscription } from '@urql/vue'
-import { Debug_RelevantRuns_SubscriptionDocument } from '@packages/app/src/generated/graphql'
+import { Debug_RelevantRuns_SubscriptionDocument, Sidebar_RelevantRuns_SubscriptionDocument } from '@packages/app/src/generated/graphql'
+import { useLoginConnectStore } from '@packages/frontend-shared/src/store/login-connect-store'
+
 import { computed } from 'vue'
 
+/**
+ * Using two different subscriptions with different names in order for urql to treat them separately.
+ *
+ * Currently, this composable is used in two places and can have two subscriptions running at one time.
+ * Without having two separately named subscriptions, urql would not tell the websocket that one of the
+ * subscriptions had ended when the component it was registered in was unmounted.
```

Comment:
This is great to have figured out. Does this mean that it is safest if, globally, no two subscriptions have the same name?

Reply:
The code generation that generates the types from the `gql` blocks in the code would yell at you if tried to create two queries/mutation/subscriptions with the same names.  In this case, it was because the original code was subscribing to the exact same GQL document at the same time.  There is probably an expected way to handle this, but I could not find one documented.  When I started going through the Urql code, I ultimately found [this code](https://github.com/urql-graphql/urql/blob/main/packages/core/src/utils/request.ts#L75-L92) where it was keying the operation by hashing it and some other stuff.  That made me think to try the different names which worked, so just went with that.